modules/rosa-sharing-vpc-cluster-creation.adoc

```diff
@@ -3,25 +3,24 @@
 // * networking/rosa-shared-vpc-config.adoc
 :_content-type: PROCEDURE
 [id="rosa-sharing-vpc-cluster-creation_{context}"]
-= Creating your cluster in a shared VPC
+= Step Four - Cluster Creator: Creating your cluster in a shared VPC
 To create a cluster in a shared VPC, complete the following step. 
 
 [NOTE]
 ====
-Installing a cluster in a shared VPC is supported only for OpenShift 4.13.9 and later.
+Installing a cluster in a shared VPC is supported only for OpenShift 4.13.10 and later.
```

Comment:
```suggestion
Installing a cluster in a shared VPC is supported only for OpenShift 4.12.34 and later 4.12.z's, and 4.13.10 and all future 4.13.z's and 4.y's.
```

We have the 4.12 backports in place, starting at 4.12.34. But I don't really have a great way to say
`4.12.34+` within 4.12 and `4.13.10+` for _all_ values of `+` 😉

Reply:
The logic is documented in https://issues.redhat.com/browse/OCM-3688

Also, why did we change `4.13.9` -> `4.13.10`? It looks like `4.13.9` was the _correct_ minimum 4.13 version